core/models.py

```diff
@@ -627,6 +628,21 @@ def json(self, host, serialize=True, include_pages=True):
             return json.dumps(j, indent=2)
         return j
 
+    @property
+    def copyright_link(self):
+        public_domain_date = datetime.date(1923,01,01)
```

Comment:
Kill the leading 0; it makes Python treat this as an octal I believe.  And while that's not a problem at the moment, it isn't necessary, and makes a dev go, "hmmm..."